storage/pyramid.go

```diff
@@ -129,9 +129,10 @@ func NewTreeEntry(pyramid *PyramidChunker) *TreeEntry {
 
 // Used by the hash processor to create a data/tree chunk and send to storage
 type chunkJob struct {
-	key      Address
-	chunk    []byte
-	parentWg *sync.WaitGroup
+	key         Address
```

Comment:
changes here are only debug, consider removing pls

Reply:
Done

Reply:
watch incorrect or unnecassary whitespace changes

Reply:
I think i refactored this.. couldn't find this anymore.